Makefile

```diff
@@ -132,6 +132,10 @@ build/addon/data/panel-bundle.js: $(shoot_panel_dependencies)
 	@mkdir -p $(@D)
 	# Save the bundle dependencies:
 	browserify --list -e ./build/addon/data/shoot-panel.js | sed "s!$(shell pwd)/!!g" > build/shoot-panel-dependencies.txt
+	cat ./build/addon/lib/shooter.js | ./bin/_fixup_panel_js > ./build/addon/lib/shooter-new.js
+	mv ./build/addon/lib/shooter-new.js ./build/addon/lib/shooter.js
```

Comment:
This feels fragile to me.  I think something like:

```
build/addon/lib/shooter.js: addon/lib/shooter.js
    @mkdir -p $(@D)
    $(BABEL) < $< | ./bin/_fixup_panel_js > $@
```

I think that will handle it well.  *Maybe* that should be a general rule for things in `build/addon/data/%.js`